### PR TITLE
add contributing.md

### PR DESCRIPTION
**Pull request by doodyparizada**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+1. Do the changes in your fork
+1. Be sure you have followed the code style for the project
+1. Make sure you add unit tests to any new code you wrote
+1. Send a pull request
+
+## How to report a bug
+
+If you find a security vulnerability, do NOT open an issue. Email security@kinecosystem.com instead.
+
+When filing an issue, make sure to answer these questions:
```
Comment (OrenZak) [inline .github/CONTRIBUTING.md:30]: why all have number `1`?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+1. What's your current environment? i.e. operating system, processor architecture, node version
```
Comment (OrenZak) [inline .github/CONTRIBUTING.md]: should be Android os version

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+1. What did you do?
```
Comment (OrenZak) [inline .github/CONTRIBUTING.md]: Steps to reproduce

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+1. What did you expect to see?
+1. What did you see instead?
+
+## Code review process
+
+The core team looks at Pull Requests on a regular basis in a bi-weekly triage meeting. After feedback has been given we expect responses within two weeks.
+
+## Code of Conduct
```
Comment (OrenZak) [inline .github/CONTRIBUTING.md:40]: should it be in a different file ?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (doodyparizada) [reply .github/CONTRIBUTING.md:30]: Markdown will convert it to relevant numbers

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (doodyparizada) [reply .github/CONTRIBUTING.md:40]: same file. This is standard

**Comment by doodyparizada** [comment]
@OrenZak please approve

**Review by OrenZak — APPROVED** [review]
🥇 Thank you!

**Pull request merged**